### PR TITLE
DataGrid: add note to `filterRow`

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Each cell in the filter row contains a magnifying glass icon, pausing on which o
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md:18]: Line 15:
What do you mean by "**pausing** on which opens a drop-down list"?
Does the drop-down list appear if you simply _hover over_ the magnifying glass icon?

File: api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Each cell in the filter row contains a magnifying glass icon, pausing on which o
 
 To make the filter row visible, assign **true** to the **filterRow**.[visible](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/filterRow/visible.md '{basewidgetpath}/Configuration/filterRow/#visible') property.
 
+[note] If you use [grouped](/Documentation/Guide/UI_Components/Lookup/Enable_Grouping/) data structure to display data in a lookup column, the **All** item in the filter row is not displayed.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md]:
```suggestion
[note] If you use a [grouped](/Documentation/Guide/UI_Components/Lookup/Enable_Grouping/) data structure to display data in a lookup column, the **All** item in the filter row is not displayed.
```

File: api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md (hunk 1 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md:18]: Yeah, hover is the right word.

Comment (albertov05) [reply api-reference/10 UI Components/GridBase/1 Configuration/filterRow/filterRow.md:18]: Okay, how about the following?

`Each cell in the filter row contains a magnifying glass icon. Hovering over this icon opens a drop-down list with filters [available for the column](/api-reference/_hidden/GridBaseColumn/filterOperations.md '{basewidgetpath}/Configuration/columns/#filterOperations').`

**Pull request merged**